app/components/admin/widget/cards/row_component.html.erb

```diff
@@ -9,6 +9,12 @@
     <%= card.link_url %>
   </td>
 
+  <% unless header_section? %>
```

Comment:
We could use the `header?(card)` helper method here, right? We could also move that method to the model :thinking:.